index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name=" viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Skeleton</title>
+</head>
+<body>
+  <div id="mobile-background">
+    <div id="Status-bar"></div>
```

Comment:
- [ ] Kindly remove the icons you added in your header section. These icons are just to show that you are working on the mobile view for phones. It's actually not part of the design 👍🏼. Kindly remove it 😄.

![image](https://user-images.githubusercontent.com/63473584/184974763-be0a332e-67db-4a56-80a0-44564d60a7f9.png)